docs/static/monitoring/monitoring-internal.asciidoc

```diff
@@ -112,9 +110,7 @@ Monitoring {ls} with internal collectors uses these components:
 
 These pieces live outside of the default Logstash pipeline in a dedicated
 monitoring pipeline. This configuration ensures that all data and processing has
-a minimal impact on ordinary Logstash processing. Existing Logstash features,
-such as the <<plugins-outputs-elasticsearch,`elasticsearch` output>>, can be
-reused to benefit from its retry policies. 
+a minimal impact on ordinary Logstash processing. 
```

Comment:
Adding some comments here for content in lines above:

[Line 26 ](https://github.com/elastic/logstash/pull/11789/files#diff-67a711d07dea3d318284287b9f7a1d06R26) - this doesn't appear to tally with the current settings for Elasticsearch, which still refer to `xpack.monitoring.collection.enabled`

[Line 39](https://github.com/elastic/logstash/pull/11789/files#diff-67a711d07dea3d318284287b9f7a1d06R39) The example URL should indicate that it is a monitoring node, rather than a prod node